bin/build.sh

```diff
@@ -265,6 +266,14 @@ if [ ${INSTALL_DEPENDENCIES} -eq 1 ] && [ "$TRAVIS" != "true" ] ; then
       sudo apt-get install -y gobjc gnustep gnustep-make gnustep-common gnustep-devel
     fi
 
+    if [ ${INSTALL_RUST} -eq 1 ] ; then
```

Comment:
This should be moved below where other languages are. Here we are only installing dependencies/packages.